commands/http/client.go

```diff
@@ -146,8 +142,15 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 }
 
 func getQuery(req cmds.Request) (string, error) {
+	// some options should be kept clientside, the server doesnt care
+	skip := map[string]bool{
+		"api": true,
+	}
```

Comment:
mmm, potentially. not sure on the placement, as this function is the only spot that should be concerned with it.